source/geometry/FloatPoint3D.ooc

```diff
@@ -29,7 +30,8 @@ FloatPoint3D: cover {
 		if (r != 0.0f)
 			r = (this z / r) clamp(-1.0f, 1.0f) acos()
 		r
-	} }
+	}}
```

Comment:
Did that change recently? We decided on one some weeks ago because magic allowed one (the one which was already there) and disallowed the other.